tests/test_marshmallow_sqlalchemy.py

```diff
@@ -227,6 +264,10 @@ def test_include_fk(self, models):
         student_fields2 = fields_for_model(models.Student, include_fk=True)
         assert 'current_school_id' in student_fields2
 
+    def test_overridden_with_fk(self, models):
+        graded_paper_fields = fields_for_model(models.GradedPaper)
```

Comment:
It might be worth explicitly passing `include_fk=False` to make sure we reach the added code, even if the default value for `include_fk` changes.